Snowflake.Data/Client/SnowflakeDbException.cs

```diff
@@ -23,11 +23,11 @@ public sealed class SnowflakeDbException : DbException
         static private ResourceManager rm = new ResourceManager("Snowflake.Data.Core.ErrorMessages",
             typeof(SnowflakeDbException).Assembly);
 
-        public string SqlState { get; private set; }
+        public new string SqlState { get; private set; }

```

Comment:
_The member 'SnowflakeDbException.SqlState' does not hide an accessible member. The new keyword is not required._
😁

Reply:
Fixed